editor/src/components/inspector/flex-section.tsx

```diff
@@ -250,7 +250,7 @@ const TemplateDimensionControl = React.memo(
               'always',
               grid.elementPath,
               PP.create('style', axis === 'column' ? 'gridTemplateColumns' : 'gridTemplateRows'),
-              printArrayGridDimensions(newDimensions),
+              printArrayGridDimensions(newDimensions, 'show-area-name'),
```

Comment:
"show" feels very wrong here, since this function is about generating the template string which goes into the actual code update. I wonder if we should split this into two helper functions, stringifyArrayGridDimensions and something else for getting the visual printed version